SA/design_concept/content/design/design_specification_application.typ

```diff
@@ -0,0 +1,63 @@
+#import "../../../style.typ": *
+#import "@preview/tablex:0.0.6": tablex, hlinex
+#let load-drawio-svg(path, ..args) = image.decode(read(path).replace("Text is not SVG - cannot display", ""), ..args)
+
+= Application
+
+In /*TODO: Insert reference after !74 merges */ the suggested concept of
+Prof. Dr. Farhad Mehta, the application style was left open as a decision.
+
+The two discussed options were:
+
+/ Explicit: Leave higher-order function values as such and apply them
+  explicitly using a dedicated application function.
+
+/ Elaborate: #[Embedd a more deeper understanding of application into the
+  language, which allows to resolve nested curried function values to
+  their arguments if necessary.]
+
+A side-by-side comparison of how double application of two `5` literals to
+an `addition` function would look like in both styles can be seen in
+@elaborate_application and @explicit_application.
+
+#columns(2)[
+#figure(
+  load-drawio-svg("../../static/design_specification_application_elaborate.svg", width: 70%),
+  caption: [Elaborate application])<elaborate_application>
+#colbreak()
+#figure(load-drawio-svg("../../static/design_specification_application_explicit.svg"),
+  caption: [Explicit application])<explicit_application>
+]
+
+After some consideration, we have decided to go with the explicit style.
```

Comment:
I wouldn't say that one is better than the other; they both 'just are'. We just decided for us to go with one particular approach

Reply:
Of these two styles, the elaborate application style was chosen over explicit application, since the elaborate style stays readable when scaling up to bigger examples, while the explicit style starts to feel overloaded.

Reply:
implemented suggestion